GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -678,7 +678,17 @@ private void TextBox_KeyDown(object sender, KeyEventArgs e)
                 // Clipboard.SetText throws exception when text is null or empty. See https://msdn.microsoft.com/en-us/library/ydby206k.aspx
                 if (!string.IsNullOrEmpty(text))
                 {
+                    // clearing text before set it back somehow cure sporadic inability to paste data in commit dialog
```

Comment:
Would not it be better to insert the clipboard text directly to the TextBox and mark the event as handled?
We could also ensure that the removing image operation is needed by investigating the clipboard data format.